plugins/ohifv3/extension-monai-label/src/components/MonaiLabelPanel.tsx

```diff
@@ -134,60 +155,133 @@ export default class MonaiLabelPanel extends Component {
 
   updateView = async (response, labels) => {
     // Process the obtained binary file from the MONAI Label server
-    console.info('These are the predicted labels')
-    console.info(labels)
-    /* debugger; */
+    console.info('These are the predicted labels');
+    console.info(labels);
+
+    // for debugging only, we should get the response from the server
+  };
+
+  handleSegLoad = async () => {
+    const response = await fetch('http://localhost:3000/pred.nrrd');
```

Comment:
as you see I'm loading the segmentation assuming it is nrrd format, you should modify this for the final code so that it reads the arrayBuffer and info json (below) from the server

Reply:
Agree. Thanks!